src/uinput.c

```diff
@@ -789,6 +789,13 @@ static void ptr_abs(int x, int y, int p) {
 		ev.code = ABS_PRESSURE;
 		ev.value = p;
 		write(d, &ev, sizeof(ev));
+
+		if (!btn_touch) {
```

Comment:
because when we have this from the device:
```

Event: time 1453994031.742062, type 3 (EV_ABS), code 57 (ABS_MT_TRACKING_ID), value 27
Event: time 1453994031.742062, type 3 (EV_ABS), code 53 (ABS_MT_POSITION_X), value 1686
Event: time 1453994031.742062, type 3 (EV_ABS), code 54 (ABS_MT_POSITION_Y), value 53
Event: time 1453994031.742062, type 1 (EV_KEY), code 330 (BTN_TOUCH), value 1
Event: time 1453994031.742062, type 3 (EV_ABS), code 0 (ABS_X), value 1686
Event: time 1453994031.742062, type 3 (EV_ABS), code 1 (ABS_Y), value 53
Event: time 1453994031.742062, -------------- EV_SYN ------------
Event: time 1453994031.784105, type 3 (EV_ABS), code 53 (ABS_MT_POSITION_X), value 1685
Event: time 1453994031.784105, type 3 (EV_ABS), code 0 (ABS_X), value 1685
Event: time 1453994031.784105, -------------- EV_SYN ------------
Event: time 1453994031.826333, type 3 (EV_ABS), code 54 (ABS_MT_POSITION_Y), value 52
Event: time 1453994031.826333, type 3 (EV_ABS), code 1 (ABS_Y), value 52
Event: time 1453994031.826333, -------------- EV_SYN ------------
Event: time 1453994031.924554, type 3 (EV_ABS), code 54 (ABS_MT_POSITION_Y), value 51
Event: time 1453994031.924554, type 3 (EV_ABS), code 1 (ABS_Y), value 51
Event: time 1453994031.924554, -------------- EV_SYN ------------
Event: time 1453994031.954656, type 3 (EV_ABS), code 57 (ABS_MT_TRACKING_ID), value -1
Event: time 1453994031.954656, type 1 (EV_KEY), code 330 (BTN_TOUCH), value 0
Event: time 1453994031.954656, -------------- EV_SYN ------------ 
```

we get this from x11vnc:

```
Event: time 1453994144.572609, type 3 (EV_ABS), code 1 (ABS_Y), value 61
Event: time 1453994144.572609, type 3 (EV_ABS), code 0 (ABS_X), value 1699
Event: time 1453994144.572609, -------------- EV_SYN ------------
Event: time 1453994144.572648, type 3 (EV_ABS), code 0 (ABS_X), value 1700
Event: time 1453994144.572648, -------------- EV_SYN ------------
Event: time 1453994144.697337, type 3 (EV_ABS), code 0 (ABS_X), value 1699
Event: time 1453994144.697337, -------------- EV_SYN ------------ 
```

We don't have btn_touch (nor pressure, hence no pen-down / up info) here; so we generate it. We've been using this patch for quite a while.